rust/src/action.rs

```diff
@@ -537,7 +538,7 @@ pub struct Remove {
     /// The path of the file that is removed from the table.
     pub path: String,
     /// The timestamp when the remove was added to table state.
-    pub deletion_timestamp: DeltaDataTypeTimestamp,
+    pub deletion_timestamp: Option<DeltaDataTypeTimestamp>,
```

Comment:
I think this is incorrectly annotated in the official delta spec, we should also send a PR to the main delta repo to correct it :)

Reply:
Good idea, will do!

Reply:
https://github.com/delta-io/delta/pull/775